### ISSUE TITLE
How to get "binary segmentation label file" and "instance segmentation label file"?

### ISSUE THREAD

**Issue by yiyichun** [MaybeShewill-CV/lanenet-lane-detection#13]
Hello, 
How can the "binary segmentation label file" and "instance segmentation label file" be obtained from the original image file and the label json file? Is there a tool to convert it? Thank you :)

**Comment by MaybeShewill-CV** [comment]
@yiyichun You can follow the instruction of the dataset to establish your own training data. It is quite simple.

**Comment by KALAIVANIRAMALINGAM** [comment]
How to generate json file?and what does hsamples in it refers?am using pretrained model of vgg

**Comment by MaybeShewill-CV** [comment]
@KALAIVANIRAMALINGAM You can get the json file follow the Tusimple dataset instruction
